integration/entity.go

```diff
@@ -27,18 +27,27 @@ type EntityMetadata struct {
 }
 
 // newLocalEntity creates unique default entity without identifier (name & type)
-func newLocalEntity(storer persist.Storer) *Entity {
+func newLocalEntity(storer persist.Storer, synchronized bool) *Entity {
```

Comment:
I'm OK with this boolean argument as long as, in future patches, this way of configuring synchronization doesn't reach the public API.